terraform/sg.tf

```diff
@@ -0,0 +1,29 @@
+# ECS Security Group 
+resource "aws_security_group" "ecs" {
+  name        = "ECS-Access"
+  description = "Manage access to ECS"
+  vpc_id      = "${aws_vpc.da-wordpress-vpc.id}"
+
+  tags = {
+    Name = "ECS-Access"
+  }
+}

```

Comment:
would be better to have the ECS in place with its SG to avoid doing this kind of things.
creating a resource that is part of a different card normally leads to resource duplication and issues.

My suggestion is to not create this resource here. Leave the EFS Security group with a restricted rule(something that wil make the communication not work) and add a comment in the ECS card to update that specific resource(you can link the line of the file.)

Reply:
It's not the perfect situation, but as ECS isn't done yet I created the ECS sg in this PR and left an explicit comment here and in the ECS card. May I leave this way?

Reply:
as I said in my previous comment, my suggestion is to not create resources not related to your card. Have the security group related to your card only.

Reply:
Done